### DISCUSSION TITLE
Developer Onboarding Funding Proposal #2

### DISCUSSION THREAD

**Discussion by piber-dev** [akash-network/community#633] [Governance Proposals]
# Developer Onboarding Funding Proposal

## Introduction

The program previously targeting marketing and education is back after two successful runs! This time, we are aiming to introduce a way to make it run year-round without breaks, improve the education, and to incentivize direct contributions to Akash.

[Details from the first proposal](https://github.com/orgs/akash-network/discussions/457)
[Presented results from the first proposal](https://github.com/orgs/akash-network/discussions/599)

## Expected outcomes

We will include 1-5 missions from each category each month from deploying, providing, marketing, and direct contribution. Our goal is to have some missions be available at all times, such as developing and deploying an app to Akash. Examples from each category includes developing & deploying an app with a new feature, creating a helpful script for providers or running a certain provider, creating an useful text/video guide, and finding and fixing errors in the documentation.

The last category, direct contribution, is aiming to increase the number of core contributors to the network and technical projects requesting funds via governance proposals. The more core contributors the network gets, the more people will be active in WGs/SIGs plus on GitHub and on other platforms. The other three are here to get more developers familiar with Akash and at least temporarily boost network statistics in provider and deployment counts.

We are aiming to have one bi-weekly meeting for the WG-call and another one for assisting users and answering technical questions. Besides this, both of us have private meetings - usually bi-weekly - talking about how to best create missions and any other details happening behind the scenes such as if we need to prepare anything or do anything before a WG-call.

Furthermore we are expecting to increase the number of users to a total of 500-750 unique users across the different categories. This is roughly double the current user count.

## Resources required

> 1x community manager (taking the lead on wg-zealy calls; writing announcements; paying out rewards; interacting & making final decisions, etc)
> 1x administrator (reviewing missions, giving technical support, creating helper scripts, finalizing details, etc)
> (+ a common collaborative effort on creating tasks and running other meetings)

For the increased amount of work from the previous proposal, we are now requesting $2000/contributor/month.

Furthermore, we are planning to use $3000 USD/month for rewards. As we already have $8k USD at the time of writing, mostly from Greg's donation, we are only going to request the difference of $4000 USD unless it changes.

## Proposal funding

We are also proposing asking for funding for at least 4 months at a time while reporting every 3 months to ensure that we can make the program run without breaks. One period can be between months 1-3 and then this will be reported on, and any changes/discussion will be made during the 4th month. The 4th month would be included in the next report and months 5 and 6 would include potential changes from the discussion on month 4. With our current system we have been unable to run the program without breaks because we plan the new periods during the breaks.

If no changes are made, we will request funding for 4 months for a total balance of $28000 USD making the total request to be approximately $20000 USD after keeping the rewards pool donation in mind.

EDIT: fixed calculations

**Comment by ghost** [comment]
Let's do this! 

We are applying the experience we got from the last campaign and improving missions instructions and submissions criteria to make sure we funnel in the best users to akash network 

Points are not going anywhere as it is a great way to incentivize interested parties and create loyalty among community members
read this article to understand more about point programs and tools https://x.com/KentonPrescott/status/1810654153049358373

**Comment by rodri-r** [comment]
Great job guys!

After seeing how you both ran the last two campaigns and the number of new users, as well as interested people, I am quite sure that this program will help to onboard both new devs and users to the network. 

I have also been talking privately to @instafinanzas and I am aware of some of the missions that will be included in this new program.  I believe they are very interesting and that they will make Akash grow.

**Comment by dominikusbrian** [comment]
With this proposal, so developer that are building on Akash can apply for rewards/reimbursement for use-case they deployed on Akash Network ? Now that the Fee Grant feature on Akash in live, it would be even more doable to have the reimbursement done through it (fee grant). 

Would love to learn more on the scheme and timeline of this campaign.

**Reply by ghost** [reply]
Thats right. 

I know about the Fee Grant module, feel free to jump in the Zealy working group call or chat to discuss more about this

**Comment by dominikusbrian** [comment]
It would be great to have on of the quest to be about deploying GPU-powered APIs on Akash.
Something like AkashChat API is one thing, but there are many other GPU-powered APIs on Akash. To give an example, there's a platform called https://replicate.com/ that aggregate huge number of great GPU-powered APIs. 

![image](https://github.com/user-attachments/assets/39937192-a959-40fb-a29f-5abf3599f70a)

**Reply by ghost** [reply]
Great! Thanks for sharing! Will get back to you on this next week

**Comment by dominikusbrian** [comment]
One small idea that could be great and suitable for Zealy quest. 

**Deploy and host your personal/portfolio website on Akash.** 

Akash especially with the upcoming console v2 and other relevant upgrades, would certainly capable of offering seamless deployment very much like Vercel. With 500+ people deploying their personal web on Akash it would be a great story to sell, and at the same time a great first-hand experience of setting simple but useful deployments on Akash. 

In the zealy submission, user can even provide their link to **_something.ingress.akash-provider.cloud_** and a _**subdomain name**_ they would love to use. and we simply provide a subdomain that connect all these contestant webpages. 

For instance this akash-zealy.cloud is just 2USD/year 
![image](https://github.com/user-attachments/assets/ce8b3493-0bf1-4f3c-9c52-9ef66e439637)

we put this domain on Cloudflare. Then when user submit their zealy quest, they have the something.ingress.akash-provider.cloud which can reveal the IP and the subdomain name (e.g., myakash), then we will add it to the cloudflare and their akash deployment will be live on 
https://myakash.akash-zealy.cloud

If the user are capable enough and willing to pay for their own domain, they can set it up on their own by buying their own domain, and setting things up accordingly.

**Comment by SophoclesSummer** [comment]
This proposal was approved on 8/8/2024 by a 85% majority. 
https://www.mintscan.io/akash/proposals/263
